frontend/src/pages/HomeRootPage.jsx

```diff
@@ -9,50 +15,47 @@ const HomeRootPage = () => {
   const location = useLocation();
   const route = location.pathname;
 
-  if (route !== '/' && route !== '/login' && route !== '/signup'){
-    
+  if (route !== "/" && route !== "/login" && route !== "/signup") {
     return (
-      <>  
-          <LoggedInHeader username={data.username} />
-          <main>
-              <Outlet />
-          </main>
+      <>
+        <LoggedInHeader username={data.username} />
+        <main>
+          <Outlet />
+        </main>
       </>
-    );  
+    );
   }
 
   return (
     <>
-        <main>
-            <Outlet />
-        </main>
+      <main>
+        <Outlet />
+      </main>
     </>
   );
 };
 
 export default HomeRootPage;
 
 export const loader = async ({ request, params }) => {
-
-  const token = localStorage.getItem('authToken');
+  const token = localStorage.getItem("authToken");
 
   const response = await fetch(`${BaseURL}/user/profile`, {
-    method: 'GET',
+    method: "GET",
```

Comment:
just jotting my thoughts down for the future, no need to do this in this PR, we could probably use the facade pattern for these fetches

Reply:
Sounds good, when we move onto the logic we can implement this easily in another file